### PR TITLE
Feature windows default policy settings

### PR DESCRIPTION
**Pull request by snkutlu**
### Description

Feature to define default policy behaviour for MS Windows Firewall 
### Check List
- [x] All tests pass. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] The CLA has been signed. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD

### CODE REVIEW

**Comment by snkutlu** [comment]
@martinb3 Do you have any estimate when we can work on this (and some more subsequent ones)? 

I will appreciate if we can get it sorted out soon.

TIA

File: test/integration/default/serverspec/windows_spec.rb [hunk 1]
```diff
@@ -2,25 +2,29 @@
 require 'spec_helper'
 
 expected_rules = [
-  %r{firewall add rule name="prepend" description="prepend" dir=in service=any protocol=tcp localip=any localport=7788 interfacetype=any remoteip=any remoteport=any action=allow},
```
Comment (martinb3) [inline test/integration/default/serverspec/windows_spec.rb]: Could you explain the removal of `service=any`?

Comment (snkutlu) [inline test/integration/default/serverspec/windows_spec.rb]: Hey @martinb3, if the rule is not bound to any service, meaning any service can use this rule, service=any should not be coded. If service=any is coded, then it is only allowed for a service with name "any".

Comment (martinb3) [inline test/integration/default/serverspec/windows_spec.rb]: Perfect. Thanks!

**Comment by martinb3** [comment]
I've rebased and squashed your commits and made them a single commit, then merged it into master as part of dd39b95a712234fcbcd6db053fde3ced6186db2d.

**Pull request closed without merging**